ddtrace/contrib/botocore/patch.py

```diff
@@ -130,7 +130,7 @@ def patched_api_call(original_func, instance, args, kwargs):
         operation = None
         if args:
             operation = args[0]
-            span.resource = "%s.%s" % (endpoint_name, operation.lower())
+            span.resource = ".".join((endpoint_name, operation.lower()))
```

Comment:
is this faster?

Reply:
It looks like joining with tuples is the fastest way to concatenate strings in Python: https://stackoverflow.com/questions/1316887/what-is-the-most-efficient-string-concatenation-method-in-python (forget about f-strings as they're not available for earlier versions of Python)

Reply:
I wonder if a comment here so someone doesn't undo this change in the future?